package.xml

```diff
@@ -13,28 +13,37 @@
   <url type="repository">https://github.com/ros-drivers/urg_node</url>
 
   <author email="chadrockey@gmail.com">Chad Rockey</author>
+  <author email="modriscoll@clearpath.ai">Mike O'Driscoll</author>
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>roscpp</build_depend>
+  <build_depend>diagnostic_msgs</build_depend>
+  <build_depend>diagnostic_updater</build_depend>
+  <build_depend>dynamic_reconfigure</build_depend>
+  <build_depend>laser_proc</build_depend>
+  <build_depend>message_generation</build_depend>
+  <build_depend>nodelet</build_depend>
   <build_depend>rosconsole</build_depend>
+  <build_depend>roscpp</build_depend>
   <build_depend>sensor_msgs</build_depend>
-  <build_depend>urg_c</build_depend>
-  <build_depend>nodelet</build_depend>
-  <build_depend>dynamic_reconfigure</build_depend>
+  <build_depend>std_msgs</build_depend>
   <build_depend>tf</build_depend>
-  <build_depend>laser_proc</build_depend>
-  <build_depend>diagnostic_updater</build_depend>
+  <build_depend>urg_c</build_depend>
 
-  <run_depend>roscpp</run_depend>
+  <run_depend>diagnostic_msgs</run_depend>
+  <run_depend>diagnostic_updater</run_depend>
+  <run_depend>dynamic_reconfigure</run_depend>
+  <run_depend>laser_proc</run_depend>
+  <run_depend>message_runtime</run_depend>
+  <run_depend>nodelet</run_depend>
   <run_depend>rosconsole</run_depend>
+  <run_depend>roscpp</run_depend>
   <run_depend>sensor_msgs</run_depend>
-  <run_depend>urg_c</run_depend>
-  <run_depend>nodelet</run_depend>
-  <run_depend>dynamic_reconfigure</run_depend>
+  <build_depend>std_msgs</build_depend>
   <run_depend>tf</run_depend>
-  <run_depend>laser_proc</run_depend>
-  <run_depend>diagnostic_updater</run_depend>
+  <run_depend>urg_c</run_depend>
+
+  <test_depend>roslint</test_depend>
```

Comment:
You'll need to switch to package format 2 for this to do the thing that you expect (don't even ask).